MatrixSDK/Crypto/Algorithms/Megolm/MXMegolmEncryption.m

```diff
@@ -103,14 +104,37 @@ - (MXHTTPOperation*)encryptEventContent:(NSDictionary*)eventContent eventType:(M
     }];
 }
 
-- (MXHTTPOperation*)ensureSessionForUsers:(NSArray<NSString*>*)users
+- (BOOL) isVerificationEvent:(MXEventTypeString) eventType eventContent:(NSDictionary*)eventContent
```

Comment:
Optionally this whole method could be moved into `MXTools`, as `[MXTools isVerificationEvent]`, which will make it easier to unit test